mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,58 @@
+import {useState} from "react";
+import styles from "src/component/select/Select.module.scss";
+
+
+// Implement an option element for a select component
+
+export interface OptionType {
+  id: string;
+  value: string;
+  text: string;
+}
+
+// Implement a simple select component
+
+export interface SelectProps {
```

Comment:
Comment should look like on strings 16-18

Reply:
done